pydy/system.py

```diff
@@ -429,9 +417,8 @@ def _args_for_gen_ode_func(self):
 
         """
 
-        args = (self.eom_method.forcing_full,
-                self.coordinates,
-                self.speeds,
+        args = (self.force,
```

Comment:
`self.force` isn't a great name here, as forces mean very specific things in multibody dynamics. This is really the right hand side of : M x' = F, which I'm at a loss for a good name. I think this is the implicit first order form of the equations of motion.